CIPs/cip-draftultrageen.md

```diff
@@ -0,0 +1,69 @@
+---
+cip: <to be assigned>
+title: Ultragreen Celo
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): <Ring 0, 1, 2, 3>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+license: Apache 2.0
+---
+
+## Simple Summary
+Celo has since its inception, in April 2020, had a focus on creating a positive impact on the plannet, becoming the first carbon-negative blockchain. This proposals wants to take that efford further and redistribute half of base fees of transactions to a new Green Fund (GF) and burn the other half, base fees are now sent fully to the Community Fund. The on-chain [Carbon Offsetting Fund](https://docs.celo.org/protocol/pos/epoch-rewards-carbon-offsetting-fund) also aims to be overhauled for a more transparent approach.
+
+## Abstract
+This proposals aims to oultine the requirements to make Celo fund on-chain ReFi initiatives with half of the base fees collected from transactions. The other half aims to improve the already deficitary tokenomics by burning it. The Carbon Offsetting Fund is meant to be transfered to an on-chain contract owned by Governance.
```

Comment:
Burning half of the base tx fees seems only weakly related to redirecting the other half to the GF and in my mind, the motivations for these two things would very different. It could make sense to expand the motivation section in this regard or consider splitting the CIP into two separate CIPs.